Sources/Plasma/PubUtilLib/plSurface/hsGMaterial.h

```diff
@@ -91,6 +92,10 @@ class hsGMaterial : public plSynchedObject
     uint32_t                  fLoadFlags;
 
     float                fLastUpdateTime;
+    
+#if PLASMA_PIPELINE_GL || PLASMA_PIPELINE_METAL
+    hsGDeviceRef*                   fDeviceRef;
+#endif
```

Comment:
I would be OK with this being unconditional.

But, for correctness....
```suggestion
#if defined(PLASMA_PIPELINE_GL) || defined(PLASMA_PIPELINE_METAL)
    hsGDeviceRef*                   fDeviceRef;
#endif
```